apps/api/src/app/events/e2e/process-subscriber.e2e.ts

```diff
@@ -65,7 +65,7 @@ describe('Trigger event - process subscriber /v1/events/trigger (POST)', functio
     await session.awaitRunningJobs(newTemplate._id);
 
     const message = await messageRepository.find({
-      _organizationId: session.organization._id,
+      _environmentId: session.environment._id,
```

Comment:
What's the reason behind this change? Not familiar with the workflow but if we are having problems when identifying the messages maybe we should have both.

Reply:
@p-fernandez We can have them both but the environment id mandatory and the reason is mainly because of the caching key which is in the following format:
{entity-name : query-params : identifier (_id or _subscriberId) : environmentId}
In the beginning, I created it with organizationId as well but it was mainly unnecessary so I remove it.

ps. it is mandatory in order to invalidate the cached data because in order to do so we need entity name, identifier, and environment.

Let me know if I answered the question you have it mind :)